examples/core/instancing/app.js

```diff
@@ -133,16 +135,26 @@ class AppAnimationLoop extends AnimationLoop {
       }
     });
 
-    const timerElement = new TimerElement(this);
+    const statsWidget = new StatsWidget(this.stats, {
+      containerStyle: 'position: absolute;top: 20px;left: 20px;'
+    });
+    statsWidget.setFormatter('CPU Time', stat => `CPU Time: ${stat.getAverageTime().toFixed(2)}`);
```

Comment:
Suggestions:

* Looks like we could have a default formatter that would get us close to this?
* And maybe extend the Stats API to offer a more custom formatter to be registered by the stats produced (luma.gl) when it first registers each stat. 

Then apps would not need to deal with formatting except when they had special requirements...

Reply:
I was thinking of maybe having a set of default formatters like "time", "framerate", "memory", etc. and then have the function argument for full control.

In an earlier iteration I had the formatter associated directly with the `Stat`, but I think I prefer having anything relating to display of a stat being in the StatsWidget.